keeper/keeper.go

```diff
@@ -54,26 +62,80 @@ func NewKeeper(
 	sb := collections.NewSchemaBuilder(storeService)
 
 	return &Keeper{
-		// stakingKeeper: sk,
 		upgradeKeeper: uk,
 
 		Validators:              collections.NewMap(sb, availblob1.ValidatorsKey, "validators", collections.StringKey, collections.StringValue),
 		ClientID:                collections.NewItem(sb, availblob1.ClientIDKey, "client_id", collections.StringValue),
-		ProvenHeight:            collections.NewItem(sb, availblob1.ProvenHeightKey, "proven_height", collections.Int64Value),
+		ProvenHeight:            collections.NewItem(sb, availblob1.ProvenHeightKey, "proven_height", collections.Uint64Value),
 		PendingBlocksToTimeouts: collections.NewMap(sb, availblob1.PendingBlocksToTimeouts, "pending_blocks_to_timeouts", collections.Int64Key, collections.Int64Value),
 		TimeoutsToPendingBlocks: collections.NewMap(sb, availblob1.TimeoutsToPendingBlocks, "timeouts_to_pending_blocks", collections.Int64Key, codec.CollValue[types.PendingBlocks](cdc)),
 
 		storeKey: key,
 
 		cdc: cdc,
 
-		publishToAvailBlockInterval: publishToAvailBlockInterval,
-		app_id:                      appId,
+		appID: appId,
 
-		unprovenBlocks: make(map[int64][]byte),
+		unprovenBlocks:              make(map[int64][]byte),
+		MaxBlocksForBlob:            20, //Todo: call this from app.go, later change to params
```

Comment:
For now, I think we can read the value from config itself(just like the way we added avail config in our previous approach). Wdyt.